node/src/runtime.rs

```diff
@@ -192,17 +192,9 @@ define_trait! {
 		}
 
 		#[cfg(feature = "composable")]
-		impl for composable_runtime {
-			fn (io, deps) {
-				io.extend_with(PabloApi::to_delegate(Pablo::new(deps.client)));
-			}
-		}
+		impl for composable_runtime {}
 
-		impl for picasso_runtime {
-			fn (io, deps) {
-				io.extend_with(PabloApi::to_delegate(Pablo::new(deps.client)));
-			}
-		}
+		impl for picasso_runtime {}
```

Comment:
hm, not sure, should if be feature gates?

Reply:
Not sure I follow, I removed the RPC from the picasso runtime since it was just a dummy value being returned anyways

Reply:
(and composable as well)